### PR TITLE
0.2.12 release notes

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR adds release notes for 0.2.12

### CODE REVIEW

**Review by jni — APPROVED** [review]

**Comment by sofroniewn** [comment]
I have now updated this with #832. @tlambert03 do you want to give this a quick look over. I've left #908 in there as mentioned which I think is fine

**Review by tlambert03 — APPROVED** [review]

**Comment by sofroniewn** [comment]
Ok I will merge this now so that I can make release candidates now that can be tested now, but we won't actually release until tomorrow and so if any @napari/core-devs have a strong preference for changes to the release notes they still have time to request them

File: docs/release/release_0_2_12.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- Implement pluggy as plugin manager (#908)
+- Allow toggle theme from GUI (#943)
+- Add "Screenshot" option to File menu (#944)
+
+Improvements
+************
+- Allow 3D point selection via API (#907)
+- Abstract the construction of view/viewermodel (#957)
+- Add show param to Viewer (#961)
+- Remove duplicated imports in setup.py (#969)
```
Comment (kne42) [inline docs/release/release_0_2_12.rst:26]: support

File: docs/release/release_0_2_12.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+************
+- Add clickable color swatch with QColorDialog (#832)
+- Implement pluggy as plugin manager (#908)
+- Allow toggle theme from GUI (#943)
+- Add "Screenshot" option to File menu (#944)
+
+Improvements
+************
+- Allow 3D point selection via API (#907)
+- Abstract the construction of view/viewermodel (#957)
```
Comment (kne42) [inline docs/release/release_0_2_12.rst:24]: support

File: docs/release/release_0_2_12.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- Add show param to Viewer (#961)
+- Remove duplicated imports in setup.py (#969)
+- Make mouse drag attributes private in qt_layerlist (#974)
+- Rename attribute on QtViewerDockWidget (#975)
```
Comment (kne42) [inline docs/release/release_0_2_12.rst:28]: rename what to what?

File: docs/release/release_0_2_12.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+- Allow LayerList to be initialized by passing in a list of layers (#979)
+- Rename list input arguments (#980)
```
Comment (kne42) [inline docs/release/release_0_2_12.rst:30]: to what?

File: docs/release/release_0_2_12.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+- Add layers.events.changed event (#982)
+
+Bugfixes
+********
+- Fix add points to empty layer (#933)
+- Fix points thumbnail (#934)
+- Fix 4D ellipses (#950)
+- Fix Points highlight bug index (Points-data6-3 test) (#972)
+- Fix viewer view_* func signature parity (#976)
```
Comment (kne42) [inline docs/release/release_0_2_12.rst:39]: support

File: docs/release/release_0_2_12.rst (hunk 1 above)
Comment (sofroniewn) [reply docs/release/release_0_2_12.rst:26]: done

File: docs/release/release_0_2_12.rst (hunk 2 above)
Comment (sofroniewn) [reply docs/release/release_0_2_12.rst:24]: done

File: docs/release/release_0_2_12.rst (hunk 3 above)
Comment (sofroniewn) [reply docs/release/release_0_2_12.rst:28]: done

File: docs/release/release_0_2_12.rst (hunk 4 above)
Comment (sofroniewn) [reply docs/release/release_0_2_12.rst:30]: done

File: docs/release/release_0_2_12.rst (hunk 5 above)
Comment (sofroniewn) [reply docs/release/release_0_2_12.rst:39]: done

**Pull request merged**